.github/actions/create-local-test-infra-resources/action.yaml

```diff
@@ -31,6 +31,8 @@ runs:
       run: make install
       working-directory: ${{ inputs.working_directory }}
       shell: bash
+      env:
+        NAMESPACE_TO_GENERATE: tnf
```

Comment:
is NAMESPACE_TO_GENERATE always equal to ${{ inputs.oc_namespace }} ? If yes, we could use this instead?

Reply:
@edcdavid You are right, I changed it.